client/components/mma/accountoverview/InAppPurchaseCard.tsx

```diff
@@ -0,0 +1,43 @@
+import { css } from '@emotion/react';
+import { from, headline, palette, space } from '@guardian/source-foundations';
+import type { AppSubscription } from '../../../../shared/mpapiResponse';
+import { InfoSummary } from '../paymentUpdate/Summary';
+import { Card } from '../shared/Card';
+
+const productTitleCss = css`
+	${headline.xxsmall({ fontWeight: 'bold' })};
+	color: ${palette.neutral[100]};
+	margin: 0;
+	max-width: 20ch;
+
+	${from.tablet} {
+		${headline.small({ fontWeight: 'bold' })};
+	}
+`;
+
+export const InAppPurchaseCard = (props: {
+	inAppPurchase: AppSubscription;
+}) => {
+	return (
+		<>
```

Comment:
I'd wrap the two components with a `<Stack>` to handle the spacing so then you don't need the fragment or the CSS override on `<InfoSummary>`.

Reply:
good suggestion 👍